peer/client.go

```diff
@@ -32,6 +33,8 @@ type NetworkClient interface {
 	// Gossip sends given gossip message to peers
 	Gossip(gossip []byte) error
 
+	GossipSpecific(gossip []byte, nodeIDs set.Set[ids.NodeID]) error
```

Comment:
Can we change the order of these parameters to match the style of the other networking functions?